.github/ISSUE_TEMPLATE/feature-request.md

```diff
@@ -1,8 +1,4 @@
----
-name: "\U0001F4A1Feature request\U0001F4A1"
-about: Open an issue only if requested by a developer
-label: enhancement
----
+
```

Comment:
Why not removing this file completely? We can recover it from history, if we really need.

Reply:
Done